flipt-client-csharp/src/FliptClient/FliptClient.csproj

```diff
@@ -0,0 +1,45 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <PackageId>Flipt.Client</PackageId>
+    <Version>0.0.1</Version>
+    <Authors>Flipt Devs (dev@flipt.io)</Authors>
+    <Company>Flipt.io</Company>
+    <Description>Flipt Client SDK</Description>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageTags>Flipt, Feature Flags</PackageTags>
+    <RepositoryUrl>https://github.com/flipt-io/flipt-client-sdks/tree/main/flipt-client-csharp</RepositoryUrl>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="../../README.md" Pack="true" PackagePath="\"/>
+  </ItemGroup>
+
+  <ItemGroup>
```

Comment:
this bit copies the libraries from where they are put in the build pipeline to where they are expected to be loaded from the SDK in https://github.com/flipt-io/flipt-client-sdks/pull/470/files#diff-e065a1af3bac696db30fc1691b298ebd9880d9cc0f1beee156926f5dfd0f58d7R68